src/geometry/GeometryGroupData.ts

```diff
@@ -1,12 +1,12 @@
 // Copyright 2019 Cognite AS
 
 import { float64Properties, float32Properties, vector3Properties, matrix4Properties,
-  primitiveName, primitiveProperties, propertyName, colorProperties, vector4Properties,
-  primitiveAttributeProperties }
+  renderedPrimitiveNameType, primitiveProperties, renderedPropertyNameType, colorProperties, vector4Properties,
```

Comment:
What's the idea behind the "rendered" naming?

Reply:
So, there are file primitives, and rendered primitives, and they're different. For example, a closedCylinder might be rendered as one cylinder and two circles